Better explain how to increase docker/wsl2 memory

mr-c opened the pull request:
I  haven't tested either the macOS nor Windows/WSL2 directions yet

On Linux systems there is no limit placed by the Docker engine on memory usage.

Sources:

https://docs.docker.com/desktop/mac/#resources
https://docs.docker.com/desktop/windows/#resources → https://docs.microsoft.com/en-us/windows/wsl/wsl-config#configure-global-options-with-wslconfig

mr-c committed:
Better explain how to increase docker/wsl2 memory

mr-c committed:
better explanation

mr-c committed:
better direct download link

douglowe reviewed (approved):
The text looks good to me. I've not checked the windows memory code, but it looks reasonable.

mr-c committed:
add nodejs install

speeds up CWL expression evaluation by not having to download and run a nodejs software container later

mr-c commented on setup.md:
```diff
@@ -152,9 +153,9 @@ data-toggle="tab">Linux</a></li>
     ~~~
     {: .language-bash}
-8. Install cwltool and graphviz using conda
+8. Install the CWL reference runner (`cwltool`), a some helper programs using conda
```
```suggestion
8. Install the CWL reference runner (`cwltool`), and some helper programs using conda
```

mr-c committed:
Update setup.md
